examples/src/components/BucketExecutorExample.jsx

```diff
@@ -0,0 +1,250 @@
+                        fontSize: 50,
+                        lineHeight: "60px",
+                        whiteSpace: "nowrap",
+                        verticalAlign: "bottom",
+                        fontWeight: 700,
+                    }}
+                >
+                    {result.data[0]}
+                </p>
+            );
+        },
+    },
+    attributeValueList: {
+        props: {
+            dimensions: [[Model.attribute(locationStateDisplayFormIdentifier)]],
+        },
+        label: "List of States",
+        // eslint-disable-next-line react/prop-types
+        DataComponent: ({ result }) => {
+            return (
+                <p
+                    className="s-bucket-executor-kpi"
+                    style={{
+                        minHeight: 60,
+                        margin: "10px 0",
+                        fontSize: 50,
+                        lineHeight: "60px",
+                        verticalAlign: "bottom",
+                        fontWeight: 700,
+                    }}
+                >
+                    {result.headerItems[0][0].map(header => header.attributeHeaderItem.name).join(", ")}
+                </p>
+            );
+        },
+    },
+    pagedExecution: {
+        props: {
+            dimensions: [
+                [Model.attribute(locationStateDisplayFormIdentifier)],
+                [Model.measure(totalSalesIdentifier)],
+            ],
+            initialPaging: {
+                offset: [0, 0],
+                limit: [1, 1],
+            },
+        },
+        label: "Paged execution",
+        // eslint-disable-next-line react/prop-types
+        DataComponent: ({ result, getPage }) => {
+            const isLastPage = result.paging.offset[0] + result.paging.count[0] >= result.paging.total[0];
+            const isFirstPage = result.paging.offset[0] === 0;
+            const prevOffset = [result.paging.offset[0] - result.paging.count[0], 0];
+            const nextOffset = [result.paging.offset[0] + result.paging.count[0], 0];
+            return (
+                <div
+                    className="s-bucket-executor-kpi"
+                    style={{
+                        display: "flex",
+                        flexDirection: "column",
+                        alignItems: "flex-start",
+                        minHeight: "3em",
+                        lineHeight: "1em",
+                        margin: "10px 0",
+                        fontSize: 30,
+                        verticalAlign: "bottom",
+                        fontWeight: 700,
+                    }}
+                >
+                    {result.headerItems[0][0].map(header => header.attributeHeaderItem.name).join(", ")}
+                    <br />
+                    {result.data[0][0]}
+                    <div>
+                        <button
+                            disabled={isFirstPage}
+                            onClick={() => getPage({ offset: prevOffset, limit: [1, 1] })}
```

Comment:
It's not really clear why limit is being set to `[1, 1]`. We should be clearer about this since this is an example.